debian/rules

```diff
@@ -105,9 +105,9 @@ endif
 	    -DBUILD_CONFIG=mysql_release \
 	    -DCONC_DEFAULT_CHARSET=utf8mb4 \
 	    -DPLUGIN_AWS_KEY_MANAGEMENT=NO \
-			-DPLUGIN_COLUMNSTORE=NO \
+	    -DPLUGIN_COLUMNSTORE=NO \
 	    -DIGNORE_AIO_CHECK=YES \
-	    -DWITH_URING=yes \
+	    -DWITH_URING=YES \
```

Comment:
This breaks `replace_uring_with_aio` which is looking for a lowercase `yes` and I suspect the cause of the failures on our builders.

Reply:
I fixed this but build still failing.. debugging.